<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>frmSettings</class>
 <widget class="QWidget" name="frmSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Acquisition Data Settings</string>
  </property>
  <widget class="QPushButton" name="botClose" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botCancel" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Data Parameters</string>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>A/D sampling rate (MHz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>80</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Time Offset (usec)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>140</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Element Space (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>20</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>No. Channels</string>
    </property>
   </widget>
   <widget class="QwtCounter" name="cNChan" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="numButtons" >
     <number>2</number>
    </property>
    <property name="basicstep" stdset="0" >
     <double>1</double>
    </property>
    <property name="minValue" >
     <double>1</double>
    </property>
    <property name="maxValue" >
     <double>192</double>
    </property>
    <property name="value" >
     <double>1</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>110</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Signal Average</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSoundV" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask" >
     <string>9999; </string>
    </property>
    <property name="maxLength" >
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>140</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sound Velocity (m/s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Steering Angle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSteer" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask" >
     <string>9999; </string>
    </property>
    <property name="maxLength" >
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAD" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength" >
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtElSpace" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength" >
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimeOff" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength" >
     <number>32767</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAVG" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>110</y>
      <width>44</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>16</number>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
